url_serde/src/lib.rs

```diff
@@ -0,0 +1,214 @@
+/*!
+
+This crate provides wrappers and convenience functions to make Url and
+Serde work hand in hand.
+
+The supported types are:
+
+* `url::Url`
+
+# How do I use a data type with a `Url` member with Serde?
+
+Use the serde attributes `deserialize_with` and `serialize_with`.
+
+```
+struct MyStruct {
+    #[serde(deserialize_with = "url_serde::deserialize",
```

Comment:
We want to keep running the rest of the tests on stable Rust on Travis. We could add a Cargo feature and enable it based on `$TRAVIS_RUST_VERSION`, but I think it’s not worth the bother. I ok with just the non-automated test that you did.